### PR TITLE
Create orders.sql

### PR DESCRIPTION
**Pull request by sbeltaine**
Creating orders.sql. This fulfills related issue #211 Create RM-Order Cluster query.

### CODE REVIEW

**Comment by sbeltaine** [comment]
Added Axel Dorrer and Scott Perry as reviewers.

**Comment by nassibnassar** [comment]
Only one reviewer needed; removed adoerrer.

File: sql/report_queries/costs/orders.sql [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        LEFT JOIN folio_reporting.po_lines_tags AS poltags ON pol.id = poltags.pol_id
+        LEFT JOIN folio_reporting.po_lines_details_subscription AS polsubdtl ON pol.id = polsubdtl.pol_id
+),
+--subquery for po_purchase_order_detail
+po_purchase_order_detail AS (
+    SELECT
+        podtl.id AS "po_detail_id",
+        podtl.order_type AS "po_order_type",
+        podtl.po_number AS "po_number",
+        podtl.date_ordered AS "po_date_ordered",
```
Comment (nassibnassar) [inline sql/report_queries/costs/orders.sql:89]: `json_extract_path_text(podtl.data, 'poDateOrdered')`

Comment (sbeltaine) [reply sql/report_queries/costs/orders.sql]: This is fixed.

**Review by scotttperry — APPROVED** [review]

**Review by nassibnassar — APPROVED** [review]

File: sql/report_queries/costs/orders.sql [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+        podtl.order_type AS "po_order_type",
+        podtl.po_number AS "po_number",
+        --podtl.po_date_ordered AS "date_ordered", using workaround below
+        JSON_EXTRACT_PATH_TEXT(podtl.data, 'dateOrdered') :: VARCHAR AS po_date_ordered,
+        podtl.workflow_status AS "po_workflow_status",
+        poacqunitids.po_acquisition_unit_id AS "po_acquisition_unit_id",
+        poacqunitids.po_acquisition_unit_name AS "po_acquisition_unit_name",
+        poonging.po_ongoing_interval AS "po_ongoing_interval",
+        poonging.po_ongoing_is_subscription AS "po_is_subscription",
+        poonging.po_ongoing_renewal_date AS "po_renewal date",
```
[the remark this answers is not in the document]
Comment (sbeltaine) [reply sql/report_queries/costs/orders.sql:97]: Okay. I see that po_renewal_date is missing an underscore as well.

**Pull request merged**